src/content/blog/drupalcon-pittsburgh.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Drupalcon Pittsburgh Wrapup"
+pubDate: 2023-06-12
+summary: Lots happened at Drupalcon Pittsburgh - Aaron Winborn Award, DDEV sustainability, getting there by bike
+author: Randy Fay
+featureImage:
+    src: /img/blog/2023/06/2023-06-05-aaron-winborn-award.jpg
+    alt: Aaron Winborn Award
+categories:
+  - Community
+---
+
+I got to go to Drupalcon NA in Pittsburgh thanks to [Plaform.sh](https://platform.sh)'s sponsorship of the DDEV project, and it was a great time.
+
+## Listening to the Community About DDEV Sustainability
+
+My most important goal was to learn from key DDEV stakeholders about how to make the DDEV project sustainable over the long term and lots of people were willing to sit down and chat with me about that; it was super useful. We talked about how to grow contributors, how to grow contributors into maintainers, and how to encourage stakeholders to properly fund the project. I'm excited about the future of DDEV and the community it depends on. Some key takeaways:
+
+* To [grow maintainers](recruiting-maintainers.md) we need to teach them about project architecture and the many components. Because DDEV is such a hodgepodge of DevOps components like Linux, Bash, Docker, Docker Compose, and Golang, folks from the PHP communities that are its biggest users may not have the confidence to dive in and improve it. But we can do some training sessions that could help with this.
```

Comment:
Best to use root-relative production URLs and include the trailing slash:

```suggestion
* To [grow maintainers](/blog/recruiting-maintainers/) we need to teach them about project architecture and the many components. Because DDEV is such a hodgepodge of DevOps components like Linux, Bash, Docker, Docker Compose, and Golang, folks from the PHP communities that are its biggest users may not have the confidence to dive in and improve it. But we can do some training sessions that could help with this.
```